<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Descripcion</class>
 <widget class="QWidget" name="Descripcion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingresa una Descripci√≥n</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>481</width>
     <height>107</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="aceptarButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>112</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelarButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>112</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
